stubs/PyMySQL/pymysql/converters.pyi

```diff
@@ -3,14 +3,14 @@ import time
 from _typeshed import Unused
 from collections.abc import Callable, Mapping, Sequence
 from decimal import Decimal
-from typing import Any, TypeVar
+from typing import Any, NoReturn, TypeVar
 from typing_extensions import TypeAlias
 
 _EscaperMapping: TypeAlias = Mapping[type[object], Callable[..., str]] | None
 _T = TypeVar("_T")
 
 def escape_item(val: object, charset: object, mapping: _EscaperMapping = None) -> str: ...
-def escape_dict(val: Mapping[str, object], charset: object, mapping: _EscaperMapping = None) -> dict[str, str]: ...
+def escape_dict(val: Mapping[str, object], charset: object, mapping: _EscaperMapping = None) -> NoReturn: ...
```

Comment:
Maybe using `@deprecated()` would be more user-friendly here.

Reply:
Well, it's not deprecated, it can't be used. Maybe the best solution is just to remove it. After all, whoever is using it, is subject to a CVE.

Reply:
Which of course means that an explicit message would be beneficial. I'm adding the `@deprecated`.